homeassistant/components/uvc/camera.py

```diff
@@ -213,7 +214,14 @@ async def stream_source(self):
         """Return the source of the stream."""
         for channel in self._caminfo["channels"]:
             if channel["isRtspEnabled"]:
-                return channel["rtspUris"][0]
+                uri = next(
+                    (
+                        uri
+                        for i, uri in enumerate(channel["rtspUris"])
+                        if re.search(self._nvr._host, uri)
```

Comment:
Do we have access to the host some other way? Otherwise we have to disable the pylint warning on this line for now.

Reply:
Not that I'm aware of - and I only used it because we already know we can talk to that up and therefore it makes sense to choose it from the list but if there's a better way to select from the list that would work too